qa-tools/gke-helm-sa.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
can you define what you mean by `helm-sa, in comments or something?

Reply:
Sure, I can add a comment for additional clarity here